lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -352,6 +367,26 @@ private void storeHmrcInheritedIdentity(
         }
     }
 
+    private boolean isHmrcInheritedIdentityWithStrongerVotPresent(
+            SignedJWT incomingInheritedIdentity, String userId) throws CredentialParseException {
+        try {
+            var vcStoreItem =
+                    verifiableCredentialService.getVcStoreItem(HMRC_MIGRATION_CRI, userId);
+            SignedJWT existingInheritedIdentity = SignedJWT.parse(vcStoreItem.getCredential());
+
+            var existingInheritedIdentityVot =
+                    userIdentityService.getVot(existingInheritedIdentity);
+            var incomingInheritedIdentityVot =
+                    userIdentityService.getVot(incomingInheritedIdentity);
+
+            return existingInheritedIdentityVot.compareTo(incomingInheritedIdentityVot) > 0;
```

Comment:
As described below, I think it might be good to make this a bit more explicit - e.g. introduce a:
```
private static final var HMRC_PROFILES_BY_STRENGTH = List.of(PCL250, PCL200);
```
or something